src/dmd/cppmangle.d

```diff
@@ -29,6 +29,7 @@ import core.stdc.stdio;
 
 import dmd.arraytypes;
 import dmd.declaration;
+import dmd.denum : isSpecialEnumIdent;
```

Comment:
Unused import.

Reply:
Surely we should have tooling that tells us this... why doesn't the compiler warn about unused imports?

Reply:
https://github.com/dlang/dmd/pull/8740 Imma getting to it.

Reply:
That is surely a question for the NG. :-)